extensions/dash/tests/test_js_utils.py

```
@@ -0,0 +1,206 @@
+    {js_file}
+
+    // Run the test code
+    const result = (() => {{
+        {js_code}
+    }})();
+
+    console.log(JSON.stringify(result));
+    """
+
+    try:
+        result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)